src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -92,6 +95,13 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
   const handleDeleteOpen = () => setDeleteModalShow(true);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const dropdownOpen = Boolean(anchorEl);
+  const { data: project, isLoading } = useSingleProject({
+    carNumber: changeRequest.wbsNum.carNumber,
+    projectNumber: changeRequest.wbsNum.projectNumber,
+    workPackageNumber: 0
+  });
+  if (!project || isLoading) return <LoadingIndicator />;
```

Comment:
Also, can we add an error condition here for useSingleProject hook, in case its hook can cause an error? 
In other words, follow the format from ProjectCardsView.tsx when it uses useSingleProject hook